crates/next-api/src/project.rs

```diff
@@ -1530,7 +1534,7 @@ impl Project {
                 self.next_config(),
                 self.execution_context(),
             ),
-            rcstr!("instrumentation-edge"),
+            LayerName::new(rcstr!("instrumentation-edge")),
```

Comment:
```suggestion
            ```suggestion
            LayerName::with_user_friendly_name(rcstr!("instrumentation"), rcstr!("Edge Instrumentation")),
```
```